main.rb

```diff
@@ -0,0 +1,101 @@
+require_relative 'app'
+require_relative 'book'
+require_relative 'person'
+require_relative 'rental'
+
+def display_menu
+  puts 'Welcome to School Library App!'
+  puts 'What would you like to do?'
+  puts '1. List all books'
+  puts '2. List all people'
+  puts '3. Create a person'
+  puts '4. Create a book'
+  puts '5. Create a rental'
+  puts '6. List all rentals for a given person id'
+  puts '7. Quit'
+end
+
+def list_all_books(app)
+  app.list_books.each do |book|
+    puts "#{book.title} by #{book.author}"
+  end
+end
+
+def list_all_people(app)
+  app.list_people.each do |person|
+    puts "#{person.name} (#{person.class})"
+  end
+end
+
+def create_person(app)
+  puts "Enter the person's name:"
+  name = gets.chomp
+  puts "Enter the person's age:"
+  age = gets.chomp.to_i
+  puts 'Do you want to create a student (1) or a teacher (2):'
+  type = gets.chomp.downcase
+  if type == '1'
+    puts "Enter the student's grade level:"
+    grade_level = gets.chomp
+    app.create_student(name, age, grade_level)
+    puts 'Student created successfully'
+  elsif type == '2'
+    puts "Enter the teacher's specialty:"
+    specialty = gets.chomp
+    app.create_teacher(name, age, specialty)
+    puts 'Teacher created successfully'
+  else
+    puts 'Invalid person type'
+  end
+end
```

Comment:
Please note when I try to create a person (Student/Teacher), I'm met with an error. Kindly fix this 👍🏽
![Screenshot from 2023-04-14 18-39-05](https://user-images.githubusercontent.com/44624138/232119740-80b1d66c-f364-41c2-adfa-c2cdc2cc6aa5.png)